lib/biran/configurinator.rb

```diff
@@ -90,7 +90,7 @@ def process_config_file(config_file)
       config_file_contents = File.read(config_file)
       config_file_contents = ERB.new(config_file_contents).result
       config_file_contents = YAML.safe_load(config_file_contents, [], [], true)
-      config_file_contents[Configurinator.env].deep_symbolize_keys!
```

Comment:
So, the class level `env` was only used here... this code change uses the instance env.... maybe we dont need the class level accessor, and just defining an instance one should be fine...